RELEASE-NOTES.md

```diff
@@ -1,5 +1,42 @@
 # Prefect Release Notes
 
+## Release 2.10.9
+
+### Worker logs can now be seen on the flow run page
+
+Workers now link relevant logs to specific flow runs, allowing you to view infrastructure-related logs on your flow run page.
+
+<img width="1294" alt="Process worker logs" src="https://github.com/PrefectHQ/prefect/assets/2586601/658c2883-69f7-4ee0-abf6-a20ee4723b3a">
+
+
+You'll see generic logs from all worker types. Integration worker implementations such as Kubernetes workers will be updated to send additional rich logs to give you insight into the behavior of flow run infrastructure.
+
+See https://github.com/PrefectHQ/prefect/pull/9496 for details.
+
+### Enhancements
+- Handle `SIGTERM` received by agent gracefully — https://github.com/PrefectHQ/prefect/pull/8691
+- Global flow and task retries and retry delay seconds — https://github.com/PrefectHQ/prefect/pull/9171
+- Add `include_submodules` option to `git_clone_project` step — https://github.com/PrefectHQ/prefect/pull/9504
```

Comment:
```suggestion
- Add support for Github Submodules in Prefect projects — https://github.com/PrefectHQ/prefect/pull/9504
```

Reply:
I prefer the previous in this case. Or:

```suggestion
- Add `include_submodules` option to `git_clone_project` step for submodule support — https://github.com/PrefectHQ/prefect/pull/9504
```

but I don't think we really need it

Reply:
I prefer the previous in this case. Or:

```suggestion
- Add `include_submodules` option to `git_clone_project` step for submodule support — https://github.com/PrefectHQ/prefect/pull/9504
```

but I don't think we really need it

Reply:
Makes sense